src/vs/platform/windows/electron-main/windowsMainService.ts

```diff
@@ -160,16 +160,12 @@ export class WindowsMainService extends Disposable implements IWindowsMainServic
 
 	openEmptyWindow(openConfig: IOpenEmptyConfiguration, options?: IOpenEmptyWindowOptions): ICodeWindow[] {
 		let cli = this.environmentMainService.args;
```

Comment:
I think the `|| undefined` can be removed in the line below?

Reply:
I wanted to restrict the remoteAuthorty: null to only the IOpenEmptyWindowOptions & IOpenWindowOptions. 
These are the parameter types for IHostService.openWindow and signal remote/local/default. 
The IHostService will then evaluate the default. On the main side, in IOpenConfiguration, used on the main side there's just remote and local (undefined).
In all other places in our code remoteAuthorty: undefined means local. Maybe we need to rename the parameter name in IOpenEmptyWindowOptions & IOpenWindowOptions to avoid confusion.